source/_components/device_tracker.fortios.markdown

```diff
@@ -0,0 +1,33 @@
+
+This component enables Home Assistant to do device tracking of devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
+
+The component rely on fortiosapi: https://pypi.org/project/fortiosapi/
+The component has been tested both on FortiGate appliance and FortiGate VM running SW FortiOS v. 6.0.x and 6.2.0.
+
+All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and WiFi devices, incl. devices detected by LLDP.
+
+The component is based on the Home Assistant `device_tracker` platform.
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: fortios
+    host: 192.168.0.1
+    token: YOUR_API_USER_KEY
+    verify_ssl: False
+```
+
+{% configuration %}
+host:
+    description: Hostname or IP address of the FortiGate
+    required: true
+    type: string
+token:
+    description: See [Fortinet Devloper Network](https://fndn.fortinet.com) for how to create a API token. Remember this component only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
+    required: true
+    type: string
+  verify_ssl:
+    description: if the SSL certificate should be verified. In most home cases users do not have a verified certificate. Default=False.
```

Comment:
```suggestion
    description: if the SSL certificate should be verified. In most home cases users do not have a verified certificate.
```

The default should not be in the description but added to the configuration list. Please add it.